app/metrics.yaml

```diff
@@ -330,6 +330,32 @@ metrics:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2019-09-01"
+  mozilla_products:
+    type: string_list
+    description: >
+     A list of all the Mozilla products installed on device
```

Comment:
This might be a pain, but can you include the list here? That way, if someone is looking to make sure that list is "complete" or needs to be updated, it's here in this documentation.